compiler/luci-interpreter/src/kernels/FullyConnected.cpp

```diff
@@ -231,5 +249,69 @@ void FullyConnected::evalHybridWI4AF32() const
     getTensorShape(output()), getTensorData<float>(output()));
 }
 
+void FullyConnected::evalHybridWU4AF32() const
+{
+  float activation_min{};
+  float activation_max{};
+  calculateActivationRange(_params.activation, &activation_min, &activation_max);
+
+  tflite::FullyConnectedParams params{};
+  params.float_activation_min = activation_min;
+  params.float_activation_max = activation_max;
+  params.weights_format = tflite::FullyConnectedWeightsFormat::kDefault;
+
+  const auto *weights_uint4 = getTensorData<uint8_t>(weights());
+  auto *weights_float = getTensorData<float>(scratch());
+  const Shape &weights_shape = weights()->shape();
+  const auto weights_scales = weights()->scales();
+  const auto weights_zero_points = weights()->zero_points();
+  const auto weights_quantized_dimension = weights()->quantized_dimension();
+
```

Comment:
FC layer's quantized dimension must be 0. PTAL https://github.com/Samsung/ONE/blob/master/compiler/luci/pass/src/QuantizationUtils.cpp#L262

```suggestion
  const auto weights_quantized_dimension = weights()->quantized_dimension();

  LUCI_INTERPRETER_CHECK(weights_quantized_dimension == 0);
```

Reply:
I think test data should be updated accordingly.